src/read.c

```diff
@@ -4350,6 +4350,17 @@ static avifResult avifDecoderPrepareSample(avifDecoder * decoder, avifDecodeSamp
     return AVIF_RESULT_OK;
 }
 
+// Returns AVIF_TRUE if the item should be skipped. Items should be skipped for one of the following reasons:
+//  * Size is 0.
+//  * Has an essential property that isn't supported by libavif.
+//  * Item is not a single image or a grid.
```

Comment:
Should `tmap` be included too?  
If so, `avifDecoderItemShouldBeSkipped()` could also be used in `avifDecoderDataFindToneMappedImageItem()`.

Reply:
sure, let me look into this in a follow up.